src/index.js

```diff
@@ -125,7 +140,10 @@ module.exports = Class.extend({
             self._serverless.cli.log('Topic ARN: ' + topicArn);
 
             // NOTE: does not support NextToken and paginating through subscriptions at this point
-            return Q.ninvoke(sns, 'listSubscriptionsByTopic', { TopicArn: topicArn });
+            return self.provider.request('SNS', 'listSubscriptionsByTopic',
+              { TopicArn: topicArn }, self._opts.stage, self._opts.region
```

Comment:
Indentation is incorrect here. Please just follow the same pattern you did in other places where the params are a separate variable so that this all fits on one line. That makes it more consistent with the rest of the code.